### DISCUSSION TITLE
talk about your stuff

### DISCUSSION THREAD

**Discussion by iggy2022** [tautcony/ISTA-Patcher#26] [General]
hello Tautcony
we talk about you on this link
bye
https://cartechnology.co.uk/showthread.php?tid=95361

**Comment by tautcony** [comment]
Thanks for letting me know, I don't have an account but I'll check it out.

I hope that more people could participate in this project and improve it.

**Reply by iggy2022** [reply]
salut
pas besoin de compte pour voir ce sujet


no account needed to view this topic
